### DISCUSSION TITLE
Add PDF button

### DISCUSSION THREAD

**Discussion by arpa42** [sproogen/modern-resume-theme#124] [General]
Hello, 
Congratulations on your theme, it's great!
I think it would be useful to have the possibility to add a button to download our resume in pdf. 
We can print the page in a pdf but it doesn't look like the website, it's not practical to read and it's not very nice.

**Comment by jeweljohnsonj** [comment]
Any improvements on this feature or any way to get a pdf format of the resume?

**Comment by Haki-Malai** [comment]
https://github.com/sproogen/modern-resume-theme/issues/163#issuecomment-1208606262
